src/main/java/com/fastcampus/projectboard/repository/ArticleCommentRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.fastcampus.projectboard.repository;
+
+import com.fastcampus.projectboard.domain.ArticleComment;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ArticleCommentRepository extends JpaRepository<ArticleComment,Long> {
```

Comment:
인텔리제이 자바 코딩 컨벤션에서는 보통 이 부분에 띄어쓰기를 적용합니다. 인텔리제이에서는 코드 서식 재지정(reformat code) 기능을 이용해 코드 포매팅 결과를 확인해 보세요.

```suggestion
public interface ArticleCommentRepository extends JpaRepository<ArticleComment, Long> {
```

Reply:
넵 그렇게하는편이 컨벤션을 지킬 수 있겠네요 수정해놓겠습니다 감사합니다.